wa_tilda_parser.py

```diff
@@ -246,6 +251,46 @@ def filter(self, message):
 dispatcher.add_handler(kasa_handler)
 
 
+# daily poll job
+def poll_cancel(update, context):
+    return
+
+def create_poll(update, context):
+    """Sends a predefined poll"""
+    poll_data = random.choice(POLLS)
+    message = context.bot.send_poll(
+        update.effective_chat.id,
+        poll_data['title'],
+        poll_data['questions'],
+        is_anonymous=True,
+        allows_multiple_answers=False,
+    )
+
+
+def callback_daily(context):
+    context.bot.send_message(chat_id=context.job.context, text='Згенерувати опитування?',
+                             reply_markup=ReplyKeyboardMarkup(BUTTONS, resize_keyboard=True, one_time_keyboard=True))
+
+def set_daily_message(update, context):
```

Comment:
would be great to have some comments of what functions are dong. that can save a lot of time for readers in future